recipes/sparse/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "sparse" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "e7961a81592e0372a21023fcb29e651787b03ac66e3fd6df81772d3d70dc25a5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - numpy >=1.14
+    - scipy >=0.19
+    - numba >=0.37
+  run:
+    - python
+    - numpy >=1.14
+    - scipy >=0.19
+    - numba >=0.37
+
+test:
+  requires:
+    - pytest
+    - pytest-cov
+    - pytest-flake8
+  commands:
+    - py.test
```

Comment:
Perhaps `py.test sparse` ?

Reply:
I may be mistaken, but I think the preference is to not run full test suites as part of a conda-forge recipe, because it eats up too many CI resources.

Reply:
See section 6.4 of the user guide: https://media.readthedocs.org/pdf/conda-forge/latest/conda-forge.pdf

Reply:
Fair enough. I'll just take out the test and we'll do the testing on our own CI.